api/src/session/session_store.ts

```diff
@@ -57,6 +57,11 @@ export class SessionStore {
     const result = await this.pool.query(q, v);
 
     const session: Session = new Session();
+
+    if (!result.rows.length) {
+      return session;
```

Comment:
is this supposed to create session if one does not exist?

Reply:
ill make it throw an error